pkg/process/utils.go

```diff
@@ -62,6 +63,20 @@ func (s *safePid) set(pid int) {
 	s.Unlock()
 }
 
+type atomicBool int32
```

Comment:
Why is this not locked by `p.mu` yes this a atomic so its thread safe but it still enables the race conditions between a status and pause call.

Reply:
Because pausing only makes sense outside of `p.mu`. Within the lock, there is only `running` and `paused` state.

Reply:
Hmm. I'll take your word for it :)